schematic/store/synapse.py

```diff
@@ -2128,27 +2145,24 @@ def _add_id_columns_to_manifest(
 
             # Rename `Uuid` column if it wasn't specified in the schema
             if col_in_dataframe("Uuid", manifest) and not uuid_col_in_schema:
-                manifest.rename(columns={"Uuid": "Id"}, inplace=True)
+                manifest = manifest.rename(columns={"Uuid": "Id"})
             # If no `Uuid` column exists or it is specified in the schema, create a new `Id` column
             else:
                 manifest["Id"] = ""
 
-        # Retrieve the ID column name (id, Id and ID) are treated the same.
-        id_col_name = [col for col in manifest.columns if col.lower() == "id"][0]
-
-        # Check if values have been added to the Id coulumn, if not add a UUID so value in the row is not blank.
+        # Fill in UUIDs in the "Id" column if missing
         for idx, row in manifest.iterrows():
-            if not row[id_col_name]:
+            if not row["Id"]:
```

Comment:
nit: "Id" could be a made a constant at the top of this python script to reduce the chance that a dev might make a mistake later on and use a different case like "id" or "ID"

Reply:
Thanks Bryan! This is a good suggestion! I added the global constants at the top of the script.